crates/unladen_swallow/tests/ds_test.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+import pandas as pd
+
+from unladen_swallow import DiGraph, Graph
+
+
+def test_numpy_graph():
+    el = np.array([[0, 1], [2, 3], [4, 1]], dtype=np.uint32)
```

Comment:
I love this! Some FWs also do construction from the transpose `el`s format here. For example in PyG:
```python
edge_index = torch.tensor([[0, 1, 1, 2],
                           [1, 0, 2, 1]], dtype=torch.long)
```
It's a bit similar to the pandas constructor below (one row for source and one for targets). Since we can just `g = Graph.from_numpy(np.transpose(transposed_el))` in any case I don't think it matters :)

Nice work :tada: